src/libecalc/common/utils/rates.py

```diff
@@ -156,6 +156,23 @@ def convert_none_to_nan(cls, v: float, field: ModelField) -> TimeSeriesValue:
             return math.nan
         return v
 
+    @validator("values", pre=True)
+    def timesteps_values_one_to_one(cls, v: list, values, field: ModelField):
```

Comment:
```suggestion
    def timesteps_values_one_to_one(cls, v: List[Any], values: Dict[str, Any], field: ModelField):
```
typehint as detailed as possible. Is the suggested change correct? I guess this is before the `v` becomes and is parsed to `TimeSeriesValue` but not sure...